Mac - disable the drag/drop handling within Scintilla. All drag/drop handling
will be handled by the "xul:scintilla" XBL binding - see bug 89902.

--- macosx/ScintillaMacOSX.cxx	2007-10-12 15:09:56.000000000 -0700
+++ macosx/ScintillaMacOSX.cxx	2007-10-12 16:04:00.000000000 -0700
@@ -13,6 +13,9 @@
 #include "UniConversion.h"
 #endif
 
+// Disable drag/drop within Scintilla itself.
+#define KOMODO_DISABLE_DRAG_DROP // KOMODO
+
 using namespace Scintilla;
 
 const CFStringRef ScintillaMacOSX::kScintillaClassID = CFSTR( "org.scintilla.scintilla" );
@@ -334,6 +337,10 @@
 }
 
 void ScintillaMacOSX::StartDrag() {
+#ifdef KOMODO_DISABLE_DRAG_DROP
+    inDragDrop = ddNone;
+    return;
+#endif
     if (sel.Empty()) return;
 
     // calculate the bounds of the selection
